asterius/src/Asterius/Resolve.hs

```diff
@@ -125,7 +127,7 @@ linkStart debug gc_sections verbose_err store root_syms export_funcs =
       | gc_sections = gcSections verbose_err store root_syms export_funcs
       | otherwise = fromCachedModule store
     merged_m1
-      | debug = addMemoryTrap merged_m0
+      | merged_m0 `deepseq` debug = addMemoryTrap merged_m0
```

Comment:
It's better to introduce a standalone binding for the forced `merged_m0` as mentioned in the earlier comment. A standalone binding means it'll show up in the profile report just like `merged_m0`, so we can check whether the full evaluation is a bottleneck.

Reply:
Ah, right. Should be fixed now :slightly_smiling_face: 

(commit 0a6bf6b was pushed by accident, I was just experimenting locally).